contracts/Staker.sol

```diff
@@ -183,7 +184,7 @@ contract Staker is Ownable, ReentrancyGuard {
     Set new emission details to the Staker or overwrite existing ones.
     This operation requires that emission schedule alteration is not locked.
 
-    @param _tokenSchedule An array of EmissionPoints defining the token schedule.
+    @param _tokenSchedule An array of EmissionTokens defining the token schedule.
```

Comment:
There's not actually a struct called `EmissionTokens`; the "Point" in `EmissionPoints` is supposed to refer to a point of change on some emissions curve.